src/Lecture1.hs

```diff
@@ -54,7 +55,10 @@ Explanation: @sumOfSquares 3 4@ should be equal to @9 + 16@ and this
 is 25.
 -}
 -- DON'T FORGET TO SPECIFY THE TYPE IN HERE
-sumOfSquares x y = error "TODO!"
+
+
+sumOfSquares :: Int -> Int -> Int
+sumOfSquares x y = (x)*(x) + (y)*(y)
```

Comment:
That's correct! And `()` are redundant here 🧹 
```suggestion
sumOfSquares x y = x * x + y * y
```

Reply:
I'll make the required changes here